test/outreach/read/sample_responses/sequnce-uses-both-since-fields.json

```diff
@@ -0,0 +1,946 @@
+{
+  "rows": 5,
```

Comment:
we don't need so many sample response files, just `sequences.json` and `sequences-using-since.json` is good enough. Also What does "sequences using both since fields" actually mean? What was the input to this file?